Src/Microsoft.Scripting/Runtime/NotNullAttribute.cs

```diff
@@ -32,7 +32,5 @@ public sealed class NotNullAttribute : Attribute {
     /// </summary>
     [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
     public sealed class NotNullItemsAttribute : Attribute {
-        public NotNullItemsAttribute() {
```

Comment:
Although unlikely, if you remove the this constructor and someone comes along and add a constructor overload (e.g. with an argument) then the default constructor will be gone.

Reply:
I think we had removed the NotNullAttribute constructor in the same file sometime ago as well, so this makes the changes consistent.